### DISCUSSION TITLE
Agda formula not autobumping

### DISCUSSION THREAD

**Discussion by pthariensflame** [Homebrew/discussions#5560] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

The `agda` formula is listed among the autobump formulas, but its [new release yesterday](https://github.com/agda/agda/releases/tag/v2.7.0) has not been picked up by the bot and I can't bump it manually because `brew bump-cask-pr` insists only the bot can do it.  How do we fix this?

**Comment by Bo98** [comment, marked as answer]
```
 ==> agda
Current formula version:  2.6.4.3
Latest livecheck version: 2.7.0
Open pull requests:       none
Closed pull requests:     none
Warning: This formula has resources that may need to be updated.
Warning: This formula has resources that may need to be updated.
==> Downloading https://github.com/agda/agda/archive/refs/tags/v2.7.0-r1.tar.gz
curl: (22) The requested URL returned error: 404
Error: Failed to download resource "agda"
Download failed: https://github.com/agda/agda/archive/refs/tags/v2.7.0-r1.tar.gz
Error: Failed to download resource "agda"
Download failed: https://github.com/agda/agda/archive/refs/tags/v2.7.0-r1.tar.gz
```

Seems to have gotten confused over the `r1` from last time because the `version` line didn't include that r1.

Looks like you'll have to bump it manually (`brew edit agda`) and open a PR.

**Reply by pthariensflame** [reply]
I have some build failures locally anyway from the included libraries; I'll wait until those get bumped in order to send a PR.
